pyvista/plotting/plotter.py

```diff
@@ -2058,11 +2058,20 @@ def reset_key_events(self):
         b_left_down_callback = lambda: self.iren.add_observer(
             'LeftButtonPressEvent', self.left_button_down
         )
+
+        def zoom_in():  # numpydoc ignore=GL08
+            self.camera.zoom(1.05)
+            self.render()
+
+        def zoom_out():  # numpydoc ignore=GL08
+            self.camera.zoom(0.95)
+            self.render()
```

Comment:
Let's combine functions into one!
```suggestion
        def zoom(self, value):
        """Set the zoom of the camera and render.

        Parameters
        ----------
        value : float or str
            Zoom of the camera. If a float, must be greater than 0. Otherwise,
            if a string, must be ``"tight"``. If tight, the plot will be zoomed
            such that the actors fill the entire viewport.
            self.camera.zoom(value)
            self.render()

        """
        self.camera.zoom(value)
        self.render()
```